docusaurus/docs-internals/editor-controls/data-controls/inputText.md

```diff
@@ -0,0 +1,44 @@
+
+Example of the inputText:
+
+![InputText](/img/data-controls/inputText.png)
+
+### Parameters
+
+|       Name       |  Type   | Default | Description                                                                                                                                                                                                                                                                      |
+| :--------------: | :-----: | :-----: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|       `id`       | string  |    -    | The identifier of the key where the inputText will save your data.                                                                                                                                                                                                               |
+|      `type`      | string  |    -    | Type should be `"inputText"` to use this control.                                                                                                                                                                                                                                |
+|  `placeholder`   | string  |    -    | The placeholder text displayed in the input field.                                                                                                                                                                                                                               |
+|     `label`      | string  |    -    | The label displayed on the left side of the control.                                                                                                                                                                                                                             |
+|    `position`    | number  |    -    | The position of the control in the toolbar.                                                                                                                                                                                                                                      |
+|    `devices`     | string  | `"all"` | Define the devices where the control will be rendered. Options: `"all"`, `"desktop"`, `"responsive"`. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices. |
+|    `disabled`    | boolean |  false  | Determines whether the control is disabled or enabled.                                                                                                                                                                                                                           |
```

Comment:
set description for disabled as in `switch` to preserve the same text in controls